cmd/ingester/flags.go

```diff
@@ -13,6 +14,7 @@ var (
 
 func init() {
 	pflag.StringVar(&listenAddr, "http.listenAddr", ":8080", "Address to listen on.")
+	pflag.Uint32Var(&logLevel, "log", 0, "Log level to use, defaults to 4 (INFO).")
```

Comment:
Looks like it defaults to 0 (PANIC) to me.... :trollface: 
Also, maybe document the log levels or link to logrus?